.github/workflows/docker-ci.yml

```
@@ -0,0 +1,124 @@
+          tags: ${{ steps.meta.outputs.TAGS }}
+          cache-from: type=registry,ref=${{ steps.meta.outputs.CACHE_REF }}
+          cache-to: type=registry,ref=${{ steps.meta.outputs.CACHE_REF }},mode=max
+
+  notify:
+    needs: build
+    if: github.event_name == 'pull_request' && github.event.action != 'closed'
+    uses: ./.github/workflows/commenter.yml
+    with:
+      issue_number: ${{ github.event.pull_request.number }}
+      marker: docker-image-comment
+      message: |
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)